modules/serverless-pingsource-kn.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * serverless/knative_eventing/serverless-pingsource.adoc
+
+[id="serverless-pingsource-kn_{context}"]
+= Using PingSource with the `kn` CLI
+
+The following sections describe how to create, verify and remove a basic PingSource using the `kn` CLI.
+
+.Prerequisites
+
+* You have Knative Serving and Eventing installed.
+* You have the `kn` CLI installed.
+
+.Procedure
+
+. To verify that `PingSource` is working, create a simple Knative
+service that dumps incoming messages to its log:
++
+----
+$ kn service create event-display --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
```

Comment:
Those long commands are hard to read. I advice to write then as:
```adoc
----
$ kn service create event-display \
    --image quay.io/openshift-knative/knative-eventing-sources-event-display:v0.13.2
----
```

Reply:
Will this still work if someone copies and pastes the command?

Reply:
Yep. Just as before, user must copy command without `$` sign. If so, then the command will work.

Only thing is to watch out that sign `\` **must** be and the end of line, not followed by any space or other sign. Then it means "do not take this next Enter as end of command".